src/uct/api/uct.h

```diff
@@ -18,6 +18,7 @@
 #include <ucs/type/status.h>
 #include <ucs/type/thread_mode.h>
 #include <ucs/type/cpu_set.h>
+#include <ucs/stats/libstats.h>
```

Comment:
pls don't include this from UCT API
need to create additional header file for just defining stats node type